core-web/apps/dotcms-ui/src/app/api/services/dot-router/dot-router.service.ts

```diff
@@ -197,6 +197,20 @@ export class DotRouterService {
         ]);
     }
 
+    /**
+     * Redirect to edit the template.
+     * If the inode is passed, load a specific version of the template
+     *
+     * @param {string} id
+     * @param {string} inode
+     * @memberof DotRouterService
+     */
+    goToEditContainer(id: string, inode?: string): void {
+        this.router.navigate([
+            inode ? `/container-new/edit/${id}/inode/${inode}` : `/container-new/edit/${id}`
```

Comment:
Let's replace the route to be `containers`.

Let's catch that as soon as possible.